biz.aQute.bndlib/src/aQute/bnd/service/Tagged.java

```diff
@@ -0,0 +1,60 @@
+package aQute.bnd.service;
+
+import static aQute.bnd.service.Tagged.RepoTags.resolve;
+import static java.util.stream.Collectors.toCollection;
+
+import java.util.Arrays;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+/**
+ * Allows to add tags to implementing classes. Originally intended for tagging
+ * repositories.
+ */
+public interface Tagged {
+
+
+	enum RepoTags {
+		/**
+		 * tag for repos which should be used for Resolving bundles. This is
+		 * also the default tag for all repos which not have specified tags
+		 * (also for bc reasons) Also see {@link Tagged#DEFAULT_REPO_TAGS}
+		 */
+		resolve
+		// add more if neded e.g. relase, baseline
+	}
+
+	/**
+	 * Each repo has by default the tag {@link RepoTags#resolve} if not tags are
+	 * set at the repo definition in build.bnd That means it is consider
+	 */
+	Set<String> DEFAULT_REPO_TAGS = Set.of(resolve.name());
```

Comment:
this should also got toe the RepositoryPlugin interface.

Reply:
Done